src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,27 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import './CommentInfo.scss';
+
+function getNamebyEmail(email) {
+  const indexAt = email.indexOf('@');
+
+  const name = email.slice(0, indexAt);
+
+  return name;
+}
+
+export const CommentInfo = ({ comment }) => (
+  <>
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{comment.name}</strong>
+
+        {getNamebyEmail(comment.email)}
```

Comment:
Should simply be `{' by '}` not the user's name
From supplied base code:
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/52002856/95c549c6-43a8-41af-b47e-c359d4a3075b)